packages/react-components/src/components/ChatMessageActionsFlyout.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ContextualMenu, DirectionalHint, IContextualMenuItem, Target } from '@fluentui/react';
+import { _formatString } from '@internal/acs-ui-common';
+import React, { useMemo } from 'react';
+import { MessageThreadStrings } from './MessageThread';
+import { chatMessageMenuStyle, menuIconStyleSet } from './styles/ChatMessageComponent.styles';
+
+interface ChatMessageActionFlyoutProps {
```

Comment:
nit: `@private` Export this as it's used to type arguments for an exported component.

Reply:
Updated